src/commands/mod.rs

```diff
@@ -0,0 +1,339 @@
+        sample1: Option<String>,
+        /// The name of the second sample to diff
+        #[arg(long)]
+        sample2: Option<String>,
+        /// The name of the output GFA file
+        #[arg(long)]
+        gfa: String,
+    },
+    /// Replace a sequence graph with a subgraph in the range of the specified coordinates
+    DeriveSubgraph {
+        /// The name of the collection to derive the subgraph from
+        #[arg(short, long)]
+        name: Option<String>,
+        /// The name of the parent sample
+        #[arg(short, long)]
+        sample: Option<String>,
+        /// The name of the new sample
+        #[arg(long)]
+        new_sample: String,
+        /// The name of the region to derive the subgraph from
+        #[arg(short, long)]
+        region: String,
+        /// Name of alternate path (not current) to use
+        #[arg(long)]
+        backbone: Option<String>,
+    },
+    /// Replace a sequence graph with subgraphs in the ranges of the specified coordinates
+    DeriveChunks {
+        /// The name of the collection to derive the subgraph from
+        #[arg(short, long)]
+        name: Option<String>,
+        /// The name of the parent sample
+        #[arg(short, long)]
+        sample: Option<String>,
+        /// The name of the new sample
+        #[arg(long)]
+        new_sample: String,
+        /// The name of the region to derive the subgraph from
+        #[arg(short, long)]
+        region: String,
+        /// Name of alternate path (not current) to use
+        #[arg(long)]
+        backbone: Option<String>,
+        /// Breakpoints to derive chunks from
+        #[arg(long)]
+        breakpoints: Option<String>,
+        /// The size of the chunks to derive
+        #[arg(long)]
+        chunk_size: Option<i64>,
+    },
+    #[command(
+        verbatim_doc_comment,
+        long_about = "Combine multiple sequence graphs into one. Example:
+    gen make-stitch --sample parent_sample --new-sample my_child_sample --regions chr1.2,chr1.3 --new-region spliced_chr1"
+    )]
+    MakeStitch {
+        /// The name of the collection to derive the subgraph from
+        #[arg(short, long)]
+        name: Option<String>,
+        /// The name of the parent sample
+        #[arg(short, long)]
+        sample: Option<String>,
+        /// The name of the new sample
+        #[arg(long)]
+        new_sample: String,
+        /// The names of the regions to combine
+        #[arg(long)]
+        regions: String,
+        /// The name of the new region
+        #[arg(long)]
+        new_region: String,
+    },
+}
+
+#[derive(Parser)]
+#[command(version, about, long_about = None, arg_required_else_help(true))]
+pub struct Cli {
+    /// The path to the database you wish to utilize
+    #[arg(short, long)]
+    pub db: Option<String>,
+    #[command(subcommand)]
+    pub command: Option<Commands>,
+}
+
+pub fn get_db_for_command(cli_context: &CliContext, operation_conn: &Connection) -> String {
```

Comment:
can we move the get_default_connection stuff this this file too?

Reply:
Yep, done